### ISSUE TITLE
[request] Start sigram minimized to system tray

### ISSUE THREAD

**Issue by rugebiker** [sialan-labs/sigram#77]
Hi, this is a request I think it would be really useful, specially if you want sigram to start when the computer boots. It is a bit annoying to start the computer and see the sigram window being openned, so it would be a better idea to have an option to start sigram minimized to the system tray (:
thanks a lot!

(using fedora)

**Comment by realbardia** [comment]
Hi...
Currently sigram use last state. If sigram close on systray, next time you open it, window will hide and if close when window is visible, next time window will show.
But it's good suggest.
We'll add a feature to turn of automatic mode and set it manualy.
Thank you

**Comment by FedeDP** [comment]
I 'm experiencing a little bug: i'm using plasma 5.1 on archlinux, and 
1) i have no systray icon, that leads to 2) 
2) i can only close Sigram by killing it. And, what's worse, if I leave it "closed" in my systray, every time i start it, it won't be visible.
I think this feature request is similar to the one above: either add a switch to let users choose whether Sigram should start visible or not, or (and this would be great!) add support for plasma 5 systray.

Thanks for your time!

**Comment by TotalKrill** [comment]
Yeah would like this too, since then i would be able to autostart it along with my desktop.

Also i am experienceing the same problem that FedeDP is having, also on arch.
